test/functional/fixtures/api/es-next/press-key/testcafe-fixtures/press-key-test.js

```diff
@@ -0,0 +1,20 @@
+// NOTE: to preserve callsites, add new tests AFTER the existing ones
+import { Hybrid } from 'testcafe';
+import { expect } from 'chai';
+
+
+fixture `PressKey`
+    .page `http://localhost:3000/api/es-next/press-key/pages/index.html`;
+
+const focusInput    = Hybrid(() => document.getElementById('input').focus());
+const getInputValue = Hybrid(() => document.getElementById('input').value);
+
+test('Press keys', async t => {
+    await focusInput();
```

Comment:
We can implement `t.focus()` but what it should do, just call javascript `element.focus()` function? We can use `t.click` instead of this.
All of our commands emulate some user actions. How a user focuses an element? He clicks on it. 
So, I'm not against to implement the `focus` command but I don't see a strong reason to do it.